configs/dumbo.yml

```diff
@@ -0,0 +1,21 @@
+details: "Drive Train only robot with 4 talon SRX in tank drive"
+cameraServer: True #
+# Start camera server for USB cameras
+#CameraServer.launch()
+
+subsystems:
+    drivetrain:
+      required: true
+      type: yaml
+      file: "talonTankDrive.yml"
```

Comment:
This file doesn't exist anymore. Should be `talonFXTankDrive.yml` or `talonSRXTankDriveSmall.yml`

Reply:
Pretty sure it's FX, but not confident enough to make the change myself

Reply:
#70 Accounts for this difference